org.eclipse.swt.gtk.linux.x86/src/org/eclipse/swt/internal/c/gtk.d

```diff
@@ -6275,6 +6281,17 @@ guint n_accels;
 _GtkAccelGroupEntry * priv_accels;
 }
 version(DYNLINK){
+    extern (C) @nogc nothrow {
+        alias TGTKgtk_icon_theme_get_default = GtkIconTheme* function();
+        alias TGTKgtk_icon_theme_lookup_by_gicon = GtkIconInfo* function(GtkIconTheme*, GIcon*, gint size, GtkIconLookupFlags);
+        alias TGTKgtk_icon_info_load_icon = GdkPixbuf* function(GtkIconInfo*, GError**);
+    }
+
+    __gshared extern(D) {
+        TGTKgtk_icon_theme_get_default gtk_icon_theme_get_default;
+        TGTKgtk_icon_theme_lookup_by_gicon gtk_icon_theme_lookup_by_gicon;
+        TGTKgtk_icon_info_load_icon gtk_icon_info_load_icon;
+    }
 mixin(gshared!(
```

Comment:
This mixin is defined in [`java.lang.util`](https://github.com/d-widget-toolkit/dwt/blob/6eed8c1/base/src/java/lang/util.d#L477), since we're dropping support for Tango, does that mean dropping support for D1 as well? Because if so, then there is less need for using the `gshared!` mixin.

Reply:
Yes. It means dropping support for D1.

But please, don't change any existing code in this PR just because of this reason.